### PR TITLE
refactor!: update sdk to use thirdparty api v1.0

### PR DESCRIPTION
**Pull request by kleyow**

### CODE REVIEW

File: src/interface/outbound/api_interfaces/openapi.d.ts [hunk 1]
```diff
@@ -61,39 +61,62 @@ export interface paths {
...
     ErrorDescription: string;
-    /** Extension key. */
+    /**
+     * ExtensionKey
+     * @description Extension key.
+     */
     ExtensionKey: string;
-    /** Extension value. */
+    /**
+     * ExtensionValue
```
Comment (elnyry-sam-k) [inline src/interface/outbound/api_interfaces/openapi.d.ts:81]:
```suggestion
     * ExtensionKey
```

**Comment by kleyow** [comment]
@elnyry-sam-k Seems like your comment disappeared and you as a reviewer poofed.

To address the comment in the email I got, openapi.d.ts is a generated file, not much I can do about that.

File: src/models/dfspTransaction.model.ts [hunk 2]
```diff
@@ -422,6 +440,7 @@ export class DFSPTransactionModel
         // This requires user input on the PISP side, so this number should be something reasonable, like 1 minute or so
         .wait(this.config.transactionRequestVerificationTimeoutSeconds * 1000)
     } catch (error) {
+      console.log(error)
```
Comment (lewisdaly) [inline src/models/dfspTransaction.model.ts:443]: any reason for keeping this in?

File: src/models/inbound/dfspLinking.model.ts [hunk 3]
```diff
@@ -257,15 +261,14 @@ export class DFSPLinkingModel
...
+    const { consentRequestsPostRequest, consentRequestsIDPutRequest, toParticipantId } = this.data
 
     // catch any unplanned errors and notify PISP
     try {
-
       if (!consentRequestsIDPutRequest) {
         throw new Error('consentRequestsIDPutRequest is null or undefined')
       }
-
+      console.log(consentRequestsIDPutRequest);
```
Comment (lewisdaly) [inline src/models/inbound/dfspLinking.model.ts:271]: Any reason to keep this in?

File: test/unit/handlers/outbound-api.test.ts [hunk 4]
```diff
@@ -418,6 +413,7 @@ describe('Outbound API routes', (): void => {
       approveResponse as unknown as Message
     ), 10)
     const response = await server.inject(request)
+    console.log(response)
```
Comment (lewisdaly) [inline test/unit/handlers/outbound-api.test.ts:416]: please remove

File: src/models/dfspTransaction.model.ts (hunk 2 above)
Comment (kleyow) [reply src/models/dfspTransaction.model.ts]: done here and throughout. this was just my lazy debugging for tests.

**Review by elnyry-sam-k — APPROVED** [review]
Thanks Kevin!

**Comment by elnyry-sam-k** [comment]
> @elnyry-sam-k Seems like your comment disappeared and you as a reviewer poofed.
> 
> To address the comment in the email I got, openapi.d.ts is a generated file, not much I can do about that.

@kleyow - that is fine; it was an issue with ordering and looking at the diff; looking at the full file it was ok

**Pull request merged**